lerna.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+  "packages" : [
+    ".",
+    "opr-devtools",
+    "opr-models",
+    "opr-core",
+    "opr-sql-database",
+    "opr-google-cloud",
+    "opr-example-server"
+  ],
+  "version": "0.0.0"
```

Comment:
This feels a little odd - OK to have no version here? Do we need to start a formal versioning process with npm?

Reply:
According to the [lerna docs](https://lerna.js.org/docs/api-reference/configuration#version), the "version" property in lerna.json is used to record the last published version of your packages when you're publishing in fixed mode. I believe this starts as 0.0.0 when you first initialize a project and only should change when you actually publish the packages.

Reply:
Cool - sg!